src/python/bot/fuzzers/syzkaller/runner.py

```diff
@@ -118,7 +120,7 @@ def fuzz(self,
           visited.add(unique_crash)
           log_lines = utils.read_data_from_file(
               os.path.join(subdir, file), eval_data=False)
-          fuzz_result.output = str(log_lines)
+          fuzz_logs += str(log_lines) + '\n'
```

Comment:
fuzz_logs should have logs from all crashes. and engine.Crash is fine as-is with log_lines from each crash.